util/scripts/install/nomos.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+NOMOSSA_PATH=/usr/local/bin/nomossa
+if [[ -f $NOMOSSA_PATH ]]; then
+    echo "[$0] Looks like nomos is already installed at $NOMOSSA_PATH"
+    echo "[$0] Doing nothing."
+    exit 0
+fi
+
+if [[ "$(type -t git)" == "" ]]; then
+    echo "[$0] Couldn't find git in your PATH!"
+    echo "[$0] Please install git before continuing."
+    exit 1
+fi
+
+# if [[ ! -d /usr/include/glib-2.0 ]]; then
+#     echo "[$0] Couldn't find glib-2.0 headers!"
+#     echo "[$0] Please install glib2-devel before continuing."
+#     exit 1
+# fi
+
+# if [[ "$(type -t gcc)" == "" ]]; then
+#     echo "[$0] Couldn't find gcc in your PATH!"
+#     echo "[$0] Please install gcc before continuing."
+#     exit 1
+# fi
+
+REPO=$(mktemp -d)
+echo "[$0] git clone https://github.com/fossology/fossology $REPO"
+git clone https://github.com/fossology/fossology $REPO || exit 1
```

Comment:
Is cloning fossology the only way to install `nomos`? Since it's so massive, it makes the install take a really long time for me, and like I mentioned above I stopped and restarted several times because I thought there was an error. If it's the only way to install it, then as a I'd like to see some indication that progress is being made and that it might take a while to finish.

Reply:
BTW, this is not me suggesting we find another way, I'm just curious as I'm unfamiliar with `nomos` and `fossology`. 😄

Reply:
That is the way. And there is no automation at this point, and because of the complexity I plan to write the docs with this as a clearly optional module.

Reply:
Gotcha. Unfortunate as it may be, we'll work with what we've got! 👍